### PR TITLE
Adds instructions for empanadas

### PR DESCRIPTION
**Pull request by luisaforozco**
Hello Pablo,
I want to contribute to your repo the instructions to make empanadas 😄 
This is a recipe from Colombia 🇨🇴

### CODE REVIEW

**Comment by agdestein** [comment]
Looks delicious!

File: recipes/empanadas.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Empanadas
+
+1. Prepare the dough.
+2. Prepare the filling.
```
Comment (agdestein) [inline recipes/empanadas.md:4]: Maybe add some salt?

Comment (APJansen) [inline recipes/empanadas.md:4]:
```suggestion

1. Prepare the dough.
2. Prepare the filling.
3. Put the filling in the dough.
```

File: recipes/empanadas.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+3. Make the empanadas.
```
Comment (PabRod) [inline recipes/empanadas.md]:
```suggestion
3. Make the empanadas.
4. Add some salt.
```

File: recipes/empanadas.md (hunk 1 above)
Comment (PabRod) [reply recipes/empanadas.md:4]: Let's leave these changes for version 2.0.

**Pull request merged**